### DISCUSSION TITLE
Welcome to Bluej-Dark-Theme Discussions!

### DISCUSSION THREAD

**Discussion by Chroxium** [Chroxium/Bluej-Themes#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dmk07official** [comment]
I recently started learning Java at school, and we’re using BlueJ. BlueJ has always had that old-school look, so I searched for themes, and yours is by far the best! Thank you for making BlueJ so much better 🔥
